---a/net/minecraft/command/CommandServerKick.java
+++b/net/minecraft/command/CommandServerKick.java
@@ -1,68 +1,55 @@
 package net.minecraft.command;
 
 import java.util.List;
+
 import net.minecraft.entity.player.EntityPlayerMP;
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.util.BlockPos;
 
-public class CommandServerKick extends CommandBase
-{
+public class CommandServerKick extends CommandBase {
+
     private static final String __OBFID = "CL_00000550";
 
-    public String getName()
-    {
+    public String getName() {
         return "kick";
     }
 
-    public int getRequiredPermissionLevel()
-    {
+    public int getRequiredPermissionLevel() {
         return 3;
     }
 
-    public String getCommandUsage(ICommandSender sender)
-    {
+    public String getCommandUsage(ICommandSender sender) {
         return "commands.kick.usage";
     }
 
-    public void execute(ICommandSender sender, String[] args) throws CommandException
-    {
-        if (args.length > 0 && args[0].length() > 1)
-        {
+    public void execute(ICommandSender sender, String[] args) throws CommandException {
+        if (args.length > 0 && args[0].length() > 1) {
             EntityPlayerMP entityplayermp = MinecraftServer.getServer().getConfigurationManager().getPlayerByUsername(args[0]);
             String s = "Kicked by an operator.";
             boolean flag = false;
 
-            if (entityplayermp == null)
-            {
+            if (entityplayermp == null) {
                 throw new PlayerNotFoundException();
-            }
-            else
-            {
-                if (args.length >= 2)
-                {
+            } else {
+                if (args.length >= 2) {
                     s = getChatComponentFromNthArg(sender, args, 1).getUnformattedText();
                     flag = true;
                 }
 
                 entityplayermp.playerNetServerHandler.kickPlayerFromServer(s);
 
-                if (flag)
-                {
-                    notifyOperators(sender, this, "commands.kick.success.reason", new Object[] {entityplayermp.getName(), s});
+                if (flag) {
+                    notifyOperators(sender, this, "commands.kick.success.reason", new Object[]{entityplayermp.getName(), s});
+                } else {
+                    notifyOperators(sender, this, "commands.kick.success", new Object[]{entityplayermp.getName()});
                 }
-                else
-                {
-                    notifyOperators(sender, this, "commands.kick.success", new Object[] {entityplayermp.getName()});
-                }
             }
-        }
-        else
-        {
+        } else {
             throw new WrongUsageException("commands.kick.usage", new Object[0]);
         }
     }
 
-    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos)
-    {
+    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
         return args.length >= 1 ? getListOfStringsMatchingLastWord(args, MinecraftServer.getServer().getAllUsernames()) : null;
     }
-}
+}
